packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -227,9 +272,30 @@ function FacetFilterControls(props: FacetFilterControlsProps) {
     },
     [facetFiltersShown],
   )
+  const combinedRangeFacetsColumnModelType = combineRangeFacetConfig
```

Comment:
Assumption here that both facet columns are of the same type.  The combination range facet filter component would not work otherwise (and since this is set in the portal configuration, it's something that we control).